integration_tests/src/main/python/hash_aggregate_test.py

```diff
@@ -116,6 +116,19 @@
     ('b', FloatGen(nullable=(True, 10.0), special_cases=[(float('nan'), 10.0)])),
     ('c', LongGen())]
 
+# grouping single-level lists
+_grpkey_list_with_non_nested_children = [[('a', RepeatSeqGen(ArrayGen(data_gen), length=3)),
```

Comment:
I am a little confused by the testing.

The last time we got in trouble because we didn't have any tests that covered the case when we needed to sort the input data.  What tests cover that case now?

Reply:
I can't find the original failure so I don't know the exact reason for the failure. 
I have added string_gen as a child element in the test to make sure we fallback to sort aggregation

Reply:
Part of me wants to say that we should test it with a version of CUDF that does not support this, but that is so old now it probably isn't feasible.